stream_srtcp.go

```diff
@@ -100,9 +100,13 @@ func (r *ReadStreamSRTCP) init(child streamSession, ssrc uint32) error {
 	r.isInited = true
 	r.isClosed = make(chan bool)
 
-	// Create a buffer and limit it to 100KB
-	r.buffer = packetio.NewBuffer()
-	r.buffer.SetLimitSize(srtcpBufferSize)
+	if r.session.bufferFactory != nil {
+		r.buffer = r.session.bufferFactory(packetio.RTCPBufferPacket, ssrc)
+	} else {
+		// Create a buffer and limit it to 100KB
```

Comment:
This comment merely paraphrases the code, and risks becoming out of sync if we change srtcpBufferSize.

Reply:
It's from original code, not mine